src/services/notification-service.js

```diff
@@ -1,6 +1,9 @@
+import moment from 'moment';
 import NotificationStore from '../stores/notification-store.js';
 import { groupNotifications } from '../utils/notification-utils.js';
 
+const MAX_NOTIFICATION_LIFETIME_IN_MONTHS = 6;
```

Comment:
We could put this in `constants.js` to make it easier to find, alongside the other such setup.

Reply:
Yep, moved!